.env.docker.dev

```diff
@@ -18,6 +18,6 @@ WEBHOOK_RETRY_INTERVAL_SECONDS=10
 HEALTH_CHECK_RETRY_INTERVAL_SECONDS=10
 FREQUENCY_TX_TIMEOUT_SECONDS=60
 DEAD_LETTER_JOB_PREFIX=dead-letter-job
-CAPACITY_LIMIT='{"type":"percentage", "value":80}'
+CAPACITY_LIMIT={"type":"percentage", "value":80}
```

Comment:
FYI, removing these enclosing quotes is only necessary if you're using the `docker run` command to start a container; it's not necessary with `docker compose`. Also it would break any package scripts that source the environment (but it's okay here, since we don't use any of those package scripts with the containerized variant)